docs/source/diagrams/D4ILLReduction-v1_wkflw.dot

```diff
@@ -0,0 +1,65 @@
+digraph Sample {
+  $global_style
+
+  subgraph decisions {
+    $decision_style
+    ExportAscii
+    CorrectDeadTime
+    CorrectBankPositions
+    CorrectRelativeEfficiencies
+    Normalise
+  }
+
+  subgraph params {
+    $param_style
+    Runs
+    BankPositionOffsetsFile
+    ZeroPositionAngle
+    EfficiencyCalibrationFile
+    NormalisationStandard
+  }
+
+  subgraph values {
+    $value_style
+    OutputWorkspace
+  }
+
+  subgraph algorithms {
+    $algorithm_style
+    LoadAndMerge
+    LoadILLDiffraction
+    ExtractMonitors
+    DeadTimeCorrection
+    RotateInstrumentComponent
+    Divide
+    Divide2 [label="Normalise"]
+    SaveAscii
```

Comment:
The SaveAscii box looks like it's not joined up properly in the workflow diagram:
![image](https://user-images.githubusercontent.com/56295817/204341234-b676887a-b141-4f91-80e7-57d657cf6992.png)

Reply:
Indeed, there is a typo, the last connection used capital letters.